support/cas-server-support-pac4j-core-clients/src/main/java/org/apereo/cas/support/pac4j/web/flow/SAML2ClientLogoutAction.java

```diff
@@ -38,7 +33,7 @@
  */
 public class SAML2ClientLogoutAction extends AbstractAction {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(SAML2ClientLogoutAction.class);
+    private static final Logger LOGGER2 = LoggerFactory.getLogger(SAML2ClientLogoutAction.class);
```

Comment:
Rename.

Reply:
This has gone with the introduction of Lombok.